iOS/MMIWG2S/Utilities/Network.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  Network.swift
+//  MMIWG2S
+//
+//  Created by Callum Johnston on 4/7/21.
+//  Copyright © 2021 Google LLC. All rights reserved.
+//
+
+import Foundation
+
+enum HttpContentType: String {
+    case json = "application/json"
+    case form = "application/x-www-form-urlencoded"
+    case all = "*/*"
+}
+
+class Network {
+    
+    /**
+     - Parameter url: URL for the POST request
+     - Parameter data: the data to be posted
```

Comment:
Will this data handle an image/file as well?

Reply:
absolutely! We can just pass in the image as a `Data` representation of the bytes.